src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberFieldSymbol.cs

```diff
@@ -571,5 +571,12 @@ internal override bool IsDefinedInSourceTree(SyntaxTree tree, TextSpan? definedW
 
             return false;
         }
+
+        internal override void AfterAddingTypeMembersChecks(ConversionsBase conversions, DiagnosticBag diagnostics)
+        {
+            bool includeNullability = DeclaringCompilation.IsFeatureEnabled(MessageID.IDS_FeatureNullableReferenceTypes);
+            Type.CheckAllConstraints(conversions.WithNullability(includeNullability), SyntaxNode.Location, diagnostics);
```

Comment:
>SyntaxNode.Location [](start = 86, length = 19)

It looks like this location includes the field initializer. Consider using `ErrorLocation` instead which should be the field identifier.

Reply:
Done